src/repl/scala/tools/nsc/interpreter/ILoop.scala

```diff
@@ -362,7 +362,8 @@ class ILoop(in0: Option[BufferedReader], protected val out: JPrintWriter) extend
   }
 
   private def changeSettings(line: String): Result = {
-    def showSettings() = for (s <- settings.userSetSettings.toSeq.sorted) echo(s.toString)
+    val s = settings
+    def showSettings() = for (s <- s.userSetSettings.sorted(Ordering.ordered[s.Setting])) echo(s.toString)
```

Comment:
There are probably no style checkers yet conceived of in the minds of undergraduates that this shadowy expression would pass, so it's lucky there are no such checkers in existence.